resdk/resources/utils.py

```diff
@@ -94,3 +96,35 @@ def endswith_colon(schema, field):
     """
     if field in schema and not schema[field].endswith(':'):
         schema[field] += ':'
+
+
+def get_resource_id(resource):
```

Comment:
This implementation makes it hard to support adding/removing data by slugs. The `get_resource_id` function does not know what resource to query.